### ISSUE TITLE
CFX does not show framerate... "No process deteced"

### ISSUE THREAD

**Issue by Calprainn** [CXWorld/CapFrameX#299]
I am trying to use CFX in combination with RTSS as an overlay for different CPU and GPU values. Everything works except the framerate. The overlay only shows 0FPS. If I activate the frame rate graph, I can at least see the frame rate provided by RTSS.

I have read in other posts that it may be because too many processes are detected. In my case, however, no running processes are detected at all. The overlay capture status only shows “Scanning for process...”. If I start any game and press F11, the voice message “No process detected” appears. Even the refresh button does not change anything.

I have tried both the installer (latest stable 1.7.4 and latest beta 1.7.5) and the portable version (1.7.4). 

![cfx-screenshot1](https://github.com/user-attachments/assets/b2ed69e3-17b2-4d00-b33b-3d043a3d8cf7)

**Comment by Taxxor90** [comment]
What games did you test this with? Are no games being detected at all or just a specific one?

**Comment by Calprainn** [comment]
I've tested a bunch of games, e.g. Dragon Age Veilguard (over EA App), Overwatch 2, the new Indy Game, Path of Exile 2 (all launched through Steam)... So I would say all games aren't detected.

**Comment by shen9175** [comment]
yeah, it seems not show any FPS related metrics. I used Nvidia card, tried recent 2-3 drivers. Don't know if it's the nvidia new driver block it or some other cause it.
![image](https://github.com/user-attachments/assets/5b8b029b-b1cb-4e29-b295-3e0e0a2ccb76)

**Comment by Calprainn** [comment]
I'm not sure whether it could have something to do with the Nvidia driver in general. I think more users would have complained about the frame rate not being displayed, especially if you have already tried the last 2-3 drivers.

**Comment by Taxxor90** [comment]
I'm using the latest Nvidia driver right now, so it's probably not that^^

**Comment by Calprainn** [comment]
In my case the problem exists since the day I'm using CFX (which is a couple of weeks now) and in this time I used two or three versions of the Nvidia driver. So I would agree with that. ;)

**Comment by Calprainn** [comment]
Me again. I figured out why CFX is not displaying my framerate... when I use the PC monitor, CFX displays the framerate normally. However, when I stream my desktop/game via Sunshine to another computer or TV (which is what I primarily do), CFX does not recognise any process and the framerate display shows 0 fps. 

Other tools also show the FPS when streaming. I have no idea if this is a bug in CFX or if CFX is simply overwhelmed with the situation of streaming the screen output?!

**Comment by Taxxor90** [comment]
So the way it works with CFX is that we tell RTSS which framerate it should show. And we do that by sending it the ID of the process that is being detected. 
I don't know how Sunshine works exactly, but it seems that PresentMon isn't able to detect the game running on your PC when you stream it. 
And when we can't detect the game, we can't tell RTSS which framerate it should show.

**Comment by Calprainn** [comment]
It would be too good to be true if I had a great tool like CFX to show my CPU and GPU values individually.

Aber ich danke dir trotzdem für seine Bemühungen! Kannst den Fall jetzt schließen. 😉

**Comment by Taxxor90** [comment]
> Would be nice if it could show active windows applications as in the Windows task manager what gets listed under "(Apps) and maybe ignore Backround processes and Windows processes

That's exactly what it does, it shows all active applications that are detected by PresentMon and the Ignore list on the right is to ignore the unwanted applications it detects
